Package.swift

```diff
@@ -23,10 +23,11 @@ var package = Package(
     .library(name: "Data", targets: ["Data"]),
     .library(name: "Storage", targets: ["Storage", "sqlcipher"]),
     .library(name: "BrowserIntentsModels", targets: ["BrowserIntentsModels"]),
-    .library(name: "BraveWidgetsModels", targets: ["BraveWidgetsModels"]),
+    .library(name: "BraveWidgets", targets: ["BraveWidgets"]),
```

Comment:
Whats the reason for renaming this? We dont actually have the widgets inside this package still?

Reply:
Ahh I can change the name back. I used to have the widgets inside this package, but I got some weird compilation errors, so I had to move it out at the last moment.

Reply:
Fixed @kylehickinson I had it as `BraveWidgets.framework` so everything was in there. I changed it back to BraveWidgetsModels and removed the dependencies on `BraveShared`